### ISSUE TITLE
Superuser - My settings

### ISSUE THREAD

**Issue by Ciberconsultant** [nirajconverthink/CIBERCONSULTORES#13]
I understand this menu is the 'shop' where the 'licences' are created and maintained to be sold through the website. In the original version, the 'my settings' menu held all the data of the superuser and we need these data to make contracts. 
If we add the 'product' list to this menu, good to know that the first 'licenses' to be sold are the different modules:
1. Accounting 
2. Tax filing
3. Property conveyance
The previous means that this list of licenses needs to be reflected when adding a new client, as these are the options they can chose from. When we add a new product, this product also needs to be shown as an option in the 'add client' menu.

**Comment by Ciberconsultant** [comment]
See comment on Master. This menu also holds certain settings. Maybe we can 'merge' the menus 'master' and 'my settings'

**Comment by nirajconverthink** [comment]
**Reply** : Currently we don’t have any privilege to add new product so we have to fix these three . while a client purchasing a license currently we are given option to select a license. How to show these products to client also as each license is already associated with a product ? can a single license number be used by different company ? In our thought the license number should be uniquely generated & can’t be used more than once.

**Comment by Ciberconsultant** [comment]
You are right but not accurate, the three licenses create in the interface of the company who purchases them certain features that are standard. 
If they purchase only license 1, they only have access to the feature of the accounting license and will not see in their interface the features of the tax filling or property conveyance license. Their clients will not see any other option than the ones given by the accounting license. 
So, for example Company 1 and  Company 2
Company 1 has the license 2 and 3: their employes can only do tax filing or conveyance, but not accounting.
Client 1A has a full package: This client can do tax filing and conveyance
Client 1B has only the license 3: This client can only do conveyance
Company 2 has license 1 and 3: their employees can only do accounting and conveyance. 
Client 2A has a full package: This client can do accounting and conveyance
Client 2B has only the license 1: This client can only do accounting

The previous means that Company 1 and 2 and their clients, cannot see any features of the licenses they have not bought in their dashboards.

But a license of superuser can be sold to many companies, and the companies can resell te licenses they bought many times.

The only differences can be the price of each license and payment options. This is up to every user (superuser and company) to establish 

The menu 'My Settings' --> 'My Settings' - 'License Configuration' is where the superuser can create variations of sales conditions of licenses, for example:
1. Gold package: all available licenses for XX dollar/month
2. Gold package: all available licenses for XX dollar/year, applying a small discount
3. License 1 for XX dollar/month
4. License 1 for XX dollar/year, applying a small discount
5. etc.

So the price and conditions to apply change, but the content of the license is always the same. In singel license, only the singel interface is activated, in a package deal, the package licenses are activated.

**Comment by nirajconverthink** [comment]
Done Please check

**Comment by Ciberconsultant** [comment]
I don't see the changes
